src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/CustomObject.txt

```diff
@@ -1,23 +1,91 @@
-﻿>> Value(Index(ParseJson("[5]"), 1))
+﻿// Value tests
+>> Value(Index(ParseJson("[5]"), 1))
 5
 
 >> Value(ParseJson("5"))
 5
 
+>> Value(ParseJson("true"))
+1
+
+>> Value(ParseJson("false"))
+0
+
+>> Value(ParseJson("""5"""))
+5
+
+>> Value(ParseJson("""a"""))
+#Error
+
+>> Value(ParseJson("null"))
+Blank()
+
+>> Value(ParseJson("{""a"": 5}").a)
+5
+
+>> Value(ParseJson("{""a"": 5}").b)
+Blank()
```

Comment:
a few more cases (to catch the various conditions in EvalVisitor)

- ParseJson("{}").a // empty record
- ParseJson("{a:null}").a.b //  coalesce, via TryGetProperty check
- ParseJson("null").a.b // getting a BlankType custom object  directly (bypassing the TryGetProperty check)

Reply:
Done